Articles/Dashboards/extending.md

```diff
@@ -0,0 +1,319 @@
+# Dashboards
+Each section of the Umbraco backoffice has its own set of default dashboards.
+
+The dashboard area of Umbraco is used to display an 'editor' for the selected item in the tree. If no item is selected, for example when the section 'first loads', then the default set of section dashboards are displayed in the dashboard area, arranged over multiple tabs.
+
+## Registering your own Dashboard
+There are two approaches to registering a custom dashboard to appear in the Umbraco Backoffice:
+
+### Registering with package.manifest
+Add a file named 'package.manifest' to the 'App_Plugins' folder, containing the following json configuration pointing to your dashboard view:
+
+```json
+{
+    "dashboards":  [
+        {
+            "alias": "myCustomDashboard",
+            "view":  "/App_Plugins/myCustom/dashboard.html",
+            "sections":  [ "content", "member", "settings" ],
+            "weight": -10
+        }
+    ]
+}
+```
+
+The section aliases can be found in the C# developer reference for [Umbraco.Cms.Core.Constants.Applications](https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Constants.Applications.html).
```

Comment:
I suspect this link shouldn't be absolute, and will need updating (but I guess at the moment we don't know what too).